Dockerfile

```diff
@@ -1,14 +1,22 @@
-FROM golang:1.17.5-alpine3.15 as builder
-RUN apk add --no-cache git ca-certificates make tzdata
-COPY . /app
-RUN cd /app && \
-    go get -d -v && \
-    CGO_ENABLED=0 GOOS=linux go build -v -a -installsuffix cgo -o prometheus_bot
+FROM golang:1.17.6-alpine3.15 as builder
+RUN apk add --update --no-cache ca-certificates tzdata
```

Comment:
```suggestion
RUN apk add --update --no-cache ca-certificates tzdata && \
```

Reply:
Thank you, but here it's not make sense since we alredy using multistage and intermediate image not relevant to result image.
As can you see this "best practive" relevant if "If you need to use a version of Docker that does not include multistage builds"
More importand to have more separate layers in build stage to mo efficient use local docker cache.
But this important only for local docker build or in CI.

Reply:
You're right, but it does not do any harm, right?

Also I maintain a fork of this repository (https://github.com/aminvakil/prometheus_bot) which uses hadolint to check dockerfile problems (which returns warning with this PR), as this repository Dockerfile is not changing very much, I have not opened a PR yet to add hadolint to check Dockerfile on every PR, but if maintainers was OK and we wanted to add some linter for basic tests, it's good not to add warnings until then.

Reply:
It's less cache efficient to do it as linter suggested.
But I'm a big fun of linters, because linters remove big part of entropy, so let it be.
Fixed, thank you. Do not forget create PR with linter.

Reply:
I moved this command in separate stage to use parallel build advantages